### PR TITLE
update usage.md

### PR DESCRIPTION
**Pull request by SauravKanchan**
# Pull Request Template

Resolves or continues [AR-4807](https://team-1624093970686.atlassian.net/browse/AR-4807).

## Changes

Update usage.md to show storage SDK can be initialized with appAddress. Dashboard won't show appId from now onwards

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: integration-tests/test.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // const { expect } = require("chai");
 const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
 // const gateway = 'https://gateway-testnet.arcana.network/';
-// const gateway = 'http://localhost:9010/';
+const gateway = 'http://localhost:9010/';
```
Comment (shaloo) [inline integration-tests/test.js:4]: Do you need to comment out this?

**Review by shaloo — APPROVED** [review]

File: integration-tests/test.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 // const { expect } = require("chai");
```
Comment (mmjee) [inline integration-tests/test.js:1]: Why are unrelated changes in this PR?
